script/prerender/manage.ts

```diff
@@ -0,0 +1,397 @@
+      {Key: 'Owner', Value: 'dante'},
+    ],
+    TemplateBody: readFile(path.join(__dirname, 'cfn.yml')),
+  }));
+}
+
+async function deleteWorkersStack() {
+  console.log('Started workers stack deletion');
+
+  return cfnClient.send(new DeleteStackCommand({StackName: WORKERS_STACK_NAME}));
+}
+
+async function prepareAndQueueBook([bookId, {defaultVersion}]: [string, {defaultVersion: string}]) {
+  // Don't have the book title yet at this point
+  console.log(`Loading book ${bookId}@${defaultVersion}`);
+
+  const book = await bookLoader(bookId, defaultVersion);
+
+  console.log(`[${book.title}] Book loaded; preparing pages`);
+
+  const pages = prepareBookPages(book).map((page) => omit('route', page.route));
+  const numBookPages = pages.length;
+  numPages += numBookPages;
+
+  // Wait until the work queue is ready
+  const workQueueUrl = await workQueuePromise;
+
+  console.log(`[${book.title}] Queuing ${numBookPages} book pages in batches of 10`);
+
+  for (let pageIndex = 0; pageIndex < pages.length; pageIndex += 10) {
+    const pageBatch = pages.slice(pageIndex, pageIndex + 10);
+
+    // If the entire request fails, this command will throw and be caught at the end of this file
+    // However, we also need to check if only some of the messages failed
+    const sendMessageBatchResult = await sqsClient.send(new SendMessageBatchCommand({
+      Entries: pageBatch.map((page, batchIndex) => ({
+        Id: batchIndex.toString(),
+        MessageBody: JSON.stringify({ payload: { page }, type: 'page' }),
+      })),
+      QueueUrl: workQueueUrl,
+    }));
+
+    const failedMessages = sendMessageBatchResult.Failed || [];
+    const numFailures = failedMessages.length;
+    if (numFailures > 0) {
+      throw new Error(`[SQS] [SendMessageBatch] Error: ${numFailures} out of ${pageBatch.length
+        } pages in a batch failed to be queued. Failures: ${JSON.stringify(failedMessages)}`);
+    }
+
+    const successfulMessages = sendMessageBatchResult.Successful || [];
+    const numSuccesses = successfulMessages.length;
+    if (numSuccesses < pageBatch.length) {
+      throw new Error(`[SQS] [SendMessageBatch] Unexpected response: received only ${numSuccesses
+        } successes out of ${pageBatch.length} pages in a batch but also no failures. Successes: ${
+        JSON.stringify(successfulMessages)}`);
+    }
+  }
+
+  console.log(`[${book.title}] All ${numBookPages} book pages queued`);
+
+  // console.log(`[${book.title}] Queuing sitemap`);
+
+  await sqsClient.send(new SendMessageCommand({
+    MessageBody: JSON.stringify({ payload: { pages, slug: book.slug }, type: 'sitemap' }),
+    QueueUrl: workQueueUrl,
+  }));
+
+  console.log(`[${book.title}] Sitemap queued`);
+
+  // Used in the sitemap index
+  return {
+    params: { book: { slug: book.slug } },
+    state: { bookUid: book.id, bookVersion: book.version },
+  };
+}
+
+async function manage() {
+  console.log('Preloading route components');
+
+  await Loadable.preloadAll();
```

Comment:
this is for the rendering, its not necessary here

Reply:
Removed